src/hotspot/share/jvmci/jvmci_globals.hpp

```diff
@@ -57,6 +58,19 @@ class fileStream;
           "Use JVMCI as the default compiler. Defaults to true if "         \
           "EnableJVMCIProduct is true.")                                    \
                                                                             \
+  product(uint, JVMCIThreadsPerNativeLibraryRuntime, 0, EXPERIMENTAL,       \
+          "Max number of threads per JVMCI native runtime. "                \
+          "Specify 0 to force use of a single JVMCI native runtime. ")      \
+          range(0, max_jint)                                                \
```

Comment:
Does it mean that value `0` has the same effect as `1`?

Reply:
No. `0` means there is a single JVMCI native runtime shared by all threads where as `1` means each thread (that accesses JVMCI) get's its own unique isolate.

Reply:
ok